feature-libs/checkout/components/components/payment-method/payment-method.component.spec.ts

```diff
@@ -397,6 +397,7 @@ describe('PaymentMethodComponent', () => {
         img: 'CREDIT_CARD',
         actions: [{ name: 'Use this payment', event: 'send' }],
         header: 'Selected',
+        label: 'paymentCard.defaultPaymentLabel',
```

Comment:
Should be transferred

Reply:
don't worry! as explained here https://sap-cx.slack.com/archives/D02A0NHD3C0/p1643298314004500?thread_ts=1643268177.004900&cid=D02A0NHD3C0